azure-pipelines/azure-pipelines.yml

```diff
@@ -300,6 +300,86 @@ stages:
               kubectl delete -f $(System.DefaultWorkingDirectory)/examples/deviceshifu/demo_device/edgedevice-socket/mock-device
             displayName: "Shifu demo device Socket E2E test"
 
+      - job: kind_e2e_test_lwm2m_with_security
```

Comment:
need to sync with main branch

Reply:
after sync, it will be retained. I deleted it directly.